src/components/pages/ManageResources/RenderResourceManagement.js

```diff
@@ -0,0 +1,262 @@
+              confirm();
+            }}
+          />
+          <div className="search-button-group">
+            <Button
+              onClick={() => {
+                confirm();
+              }}
+              type="primary"
+              icon={<SearchOutlined />}
+            >
+              Search
+            </Button>
+            <Button
+              onClick={() => {
+                clearFilters();
+              }}
+              type="danger"
+            >
+              Reset
+            </Button>
+          </div>
+        </div>
+      );
+    },
+    filterIcon: () => {
+      return <SearchOutlined className="anticon-search-main" />;
+    },
+    onFilter: (value, record) => {
+      return record.assigned_to.toLowerCase().includes(value.toLowerCase());
+    },
+  },
+];
+
+const data = [
+  {
+    key: 1,
+    category: 'Books',
+    resource_name: 'JS Book',
+    assigned_to: 'Bob',
+  },
+  {
+    key: 2,
+    category: 'Computer',
+    resource_name: 'MacBook',
+    assigned_to: 'John',
+  },
+];
+
+const formItemLayout = {
+  labelCol: {
+    sm: {
+      span: 6,
+    },
+  },
+  wrapperCol: {
+    sm: {
+      span: 16,
+    },
+  },
+};
+
+const tabList = [
+  {
+    key: 'request',
+    tab: 'Request a resource',
+  },
+  {
+    key: 'assigned',
+    tab: 'Assigned Resources',
+  },
+];
+
+const contentList = {
+  request: (
+    <Form
+      {...formItemLayout}
+      style={{ display: 'flex', flexDirection: 'column' }}
+    >
+      <p style={{ fontWeight: 'bold', paddingLeft: '1.3rem' }}>
+        Please fill this information to request a resource
+      </p>
+      <Form.Item
+        name="resource_name"
+        label="Resource Name"
+        rules={[{ required: true, message: 'Please input a resource name' }]}
+      >
+        <Input />
+      </Form.Item>
+      <Form.Item
+        name="category"
+        label="Category"
+        rules={[{ required: true, message: 'Please input a category name' }]}
+      >
+        <Input></Input>
+      </Form.Item>
+      <Form.Item
+        name="pertains_to"
+        label="Pertains to"
+        rules={[{ required: true, message: 'Please input a mentee name' }]}
+      >
+        <Input />
+      </Form.Item>
+      <Form.Item name="" label="Message">
+        <TextArea />
+      </Form.Item>
+      <Form.Item style={{ alignSelf: 'center' }}>
+        <Button>Submit request</Button>
+      </Form.Item>
+    </Form>
+  ),
+  assigned: <Table style={{}} columns={columns} dataSource={data} />,
+};
+
+export const RenderResourceManagement = () => {
+  const [activeTabKey, setActiveTabKey] = useState('request');
+  const { lg } = useBreakpoint();
+
+  const onTabChange = key => {
+    setActiveTabKey(key);
+  };
+
+  return (
+    <div>
+      {lg ? (
+        <Card
+          tabList={tabList}
+          activeTabKey={activeTabKey}
+          onTabChange={key => {
+            onTabChange(key);
+          }}
+          style={{
+            width: '40%',
```

Comment:
I strongly suggest changing this value to either a larger or non percentage-based unit, such as 40rem; with window sizes under 1200px, a 40% width causes row labels to become unreadable (tucked underneath the table, itself). In testing, a width of 40rem produced roughly the same appearance as in your video demonstration.

Reply:
Awesome, thanks Lance. I thought I had changed the value sooner, so I definitely appreciate you pointing that out.